### ISSUE TITLE
Aspect Ratio support.

### ISSUE THREAD

**Issue by mgrazianoc** [ujizin/Camposer#9]
**Is your feature request related to a problem? Please describe.**
No.

**Describe the solution you'd like**
ImageCapture use case with Aspect Ratio support.

**Describe alternatives you've considered**
Most photo applications needs to control the aspect ratio of the generated image. The default available ones for the ImageCapture use case are `RATIO_4_3` and `RATIO_16_9`.

**Are you willing to provide a PR?**
Yes.

**Comment by lectricas** [comment]
Any progress on this?

**Comment by ujizin** [comment]
Hey @lectricas, yes! It's already implemented in version `0.2.0`, you can check out in this [pull request](https://github.com/ujizin/Camposer/pull/12)

Here's an example:

```Kotlin
CameraPreview(
  // ...
  imageCaptureTargetSize = remember { ImageTargetSize(AspectRatio.RATIO_16_9) }, // or AspectRatio.RATIO_4_3
)
```
